test/helpers/position_assertions.cc

```diff
@@ -51,6 +51,7 @@ const UnorderedMap<
         {"assert-fast-path", FastPathAssertion::make},
         {"assert-slow-path", BooleanPropertyAssertion::make},
         {"hover", HoverAssertion::make},
+        {"hover-multiline", HoverMultilineAssertion::make},
```

Comment:
Double checking this name makes the most sense?

Reply:
What do you think about `hover-line` or `hover-at-line`? (Given the name I thought that a single hover-multiline assertion would accept a multiline string somehow.)